LaunchFiles totaly makeover - `mock_hardware` & `launch_rviz` params (#9)
AGH-CEAI/aegis_ros: aegis_control/README.md

# aegis_control

> This package contains the [ros2_control](https://control.ros.org/) launch files of the Aegis robot station, which uses configuration files from the `aegis_description` package.

Reviewer: ```suggestion
This package contains the [ros2_control](https://control.ros.org/) launch files for the Aegis robot station, which incorporate configuration files from the `aegis_description` package.
```

## Launch files

```
aegis_control/
├── launch
│   ├── start_drivers.launch.py
│   └── ur_driver.launch.py
```

| File                                                        | Description                                                                                                                           |
| ----------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |

> | [start_drivers.launch.py](./launch/start_drivers.launch.py) | The main launch file to run whole Aegis' ros2_control stack.                                                                          |

Reviewer: ```suggestion
| [start_drivers.launch.py](./launch/start_drivers.launch.py) | The main launch file to run the entire Aegis' `ros2_control` stack.                                                                          |
```

> | [ur_driver.launch.py](./launch/ur_driver.launch.py)         | Launches nodes from the [ur_robot_driver](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver) to control the UR5e robot. |

Author: Add development node regarding the `ur_robot_driver` 2.5.1, which requires their custom `control_manager` node for the real hardware usage.
